airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionTransformationTab/DbtCloudTransformationsCard.tsx

```diff
@@ -0,0 +1,172 @@
+import { faPlus, faXmark } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
+import { Field, Form, Formik, FieldArray, FieldProps } from "formik";
+import { Link } from "react-router-dom";
+import { array, object, number } from "yup";
+
+import { Button } from "components/ui/Button";
+import { Card } from "components/ui/Card";
+import { Input } from "components/ui/Input";
+
+import { WebBackendConnectionRead } from "core/request/AirbyteClient";
+import { useCurrentWorkspace } from "hooks/services/useWorkspace";
+import { DbtCloudJob, useDbtIntegration } from "packages/cloud/services/dbtCloud";
+import { RoutePaths } from "pages/routePaths";
+
+import styles from "./DbtCloudTransformationsCard.module.scss";
+
+const dbtCloudJobListSchema = object({
+  jobs: array().of(
+    object({
+      account: number().required().positive().integer(),
+      job: number().required().positive().integer(),
+    })
+  ),
+});
+
+export const DbtCloudTransformationsCard = ({ connection }: { connection: WebBackendConnectionRead }) => {
+  // Possible render paths:
+  // 1) IF the workspace has no dbt cloud account linked
+  //    THEN show "go to your settings to connect your dbt Cloud Account" text
+  //    and the "Don't have a dbt account?" hero/media element
+  // 2) IF the workspace has a dbt cloud account linked...
+  //   2.1) AND the connection has no saved dbt jobs (cf: operations)
+  //        THEN show empty jobs list and the "+ Add transformation" button
+  //   2.2) AND the connection has saved dbt jobs
+  //        THEN show the jobs list and the "+ Add transformation" button
+
+  const { hasDbtIntegration, saveJobs, dbtCloudJobs } = useDbtIntegration(connection);
+  const onSubmit = ({ jobs }: { jobs: DbtCloudJob[] }) => {
+    saveJobs(jobs);
+  };
+
+  return (
+    <Formik // TODO extract to parent component, see if that helps with input focus issues
+      onSubmit={onSubmit}
+      initialValues={{ jobs: dbtCloudJobs }}
+      validationSchema={dbtCloudJobListSchema}
+      render={({ values, isValid }) => {
+        return (
+          <Form className={styles.jobListForm}>
+            <FieldArray
+              name="jobs"
+              render={({ remove, push }) => {
+                return (
+                  <Card
+                    title={
+                      <span className={styles.jobListTitle}>
+                        Transformations
```

Comment:
I guess it's already on your radar, but these components are all still using hard-coded strings instead of `<FormattedMessage>`. This one I would fix before merging, lest we forget.